ERROR: Could not build wheels for deepmd-kit, which is required to install pyproject.toml-based projects

### Bug summary

Error occurs when using "pip install ." to install the DeepMD-kit under the directory of source code

### DeePMD-kit Version

N/A

### TensorFlow Version

Tried 2.9, 2.6

### How did you download the software?

Built from source

### Input Files, Running Commands, Error Log, etc.

N/A

### Steps to Reproduce

Following the tutorial here: https://docs.deepmodeling.com/projects/deepmd/en/master/install/install-from-source.html. The error arises when I try to install DeepMD-kit

### Further Information, Files, and Links

_No response_

Processing /nobackup1c/users/zhenzey/DeepMD/deepmd-kit
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
Requirement already satisfied: numpy in /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages (from deepmd-kit==2.1.5.dev0+gc393392.d20220910) (1.22.3)
Requirement already satisfied: dargs>=0.2.6 in /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages (from deepmd-kit==2.1.5.dev0+gc393392.d20220910) (0.2.6)
Requirement already satisfied: wcmatch in /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages (from deepmd-kit==2.1.5.dev0+gc393392.d20220910) (8.2)
Requirement already satisfied: scipy in /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages (from deepmd-kit==2.1.5.dev0+gc393392.d20220910) (1.7.3)
Requirement already satisfied: packaging in /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages (from deepmd-kit==2.1.5.dev0+gc393392.d20220910) (21.3)
Requirement already satisfied: python-hostlist>=1.21 in /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages (from deepmd-kit==2.1.5.dev0+gc393392.d20220910) (1.21)
Requirement already satisfied: pyyaml in /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/PyYAML-6.0-py3.10-linux-x86_64.egg (from deepmd-kit==2.1.5.dev0+gc393392.d20220910) (6.0)
Requirement already satisfied: h5py in /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages (from deepmd-kit==2.1.5.dev0+gc393392.d20220910) (3.7.0)
Requirement already satisfied: pyparsing!=3.0.5,>=2.0.2 in /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages (from packaging->deepmd-kit==2.1.5.dev0+gc393392.d20220910) (3.0.9)
Requirement already satisfied: bracex>=2.1.1 in /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages (from wcmatch->deepmd-kit==2.1.5.dev0+gc393392.d20220910) (2.1.1)
Building wheels for collected packages: deepmd-kit
  Building wheel for deepmd-kit (pyproject.toml) ... error
  error: subprocess-exited-with-error
  
  × Building wheel for deepmd-kit (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [332 lines of output]
      Not searching for unused variables given on the command line.
      -- The C compiler identification is GNU 4.8.5
      -- Detecting C compiler ABI info
      -- Detecting C compiler ABI info - done
      -- Check for working C compiler: /bin/cc - skipped
      -- Detecting C compile features
      -- Detecting C compile features - done
      -- The CXX compiler identification is GNU 4.8.5
      -- Detecting CXX compiler ABI info
      -- Detecting CXX compiler ABI info - done
      -- Check for working CXX compiler: /bin/c++ - skipped
      -- Detecting CXX compile features
      -- Detecting CXX compile features - done
      -- Configuring done
      -- Generating done
      -- Build files have been written to: /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/_cmake_test_compile/build
      -- The C compiler identification is GNU 4.8.5
      -- The CXX compiler identification is GNU 4.8.5
      -- Detecting C compiler ABI info
      -- Detecting C compiler ABI info - done
      -- Check for working C compiler: /bin/cc - skipped
      -- Detecting C compile features
      -- Detecting C compile features - done
      -- Detecting CXX compiler ABI info
      -- Detecting CXX compiler ABI info - done
      -- Check for working CXX compiler: /bin/c++ - skipped
      -- Detecting CXX compile features
      -- Detecting CXX compile features - done
      -- Found Git: /bin/git (found version "1.8.3.1")
      -- Supported model version: 1.1
      -- Will not build nv GPU support
      -- Will not build AMD GPU support
      -- Disabled cpp interface build, looking for tensorflow_framework
      -- Found TensorFlow: /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include, , /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/libtensorflow_framework.so.2  in /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow;/home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/../tensorflow_core;/home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow;/home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/../tensorflow_core;/usr/;/usr/local/ (found version "2.9.0")
      -- Performing Test CMAKE_HAVE_LIBC_PTHREAD
      -- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
      -- Looking for pthread_create in pthreads
      -- Looking for pthread_create in pthreads - not found
      -- Looking for pthread_create in pthread
      -- Looking for pthread_create in pthread - found
      -- Found Threads: TRUE
      -- Automatically determined OP_CXX_ABI=1
      -- Set GLIBCXX_USE_CXX_ABI=1 when compiling ops
      -- Found OpenMP_C: -fopenmp (found version "3.1")
      -- Found OpenMP_CXX: -fopenmp (found version "3.1")
      -- Found OpenMP: TRUE (found version "3.1")
      -- /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/libtensorflow_framework.so.2
      -- Configuring done
      -- Generating done
      CMake Warning:
        Manually-specified variables were not used by the project:
      
          PYTHON_INCLUDE_DIR
          PYTHON_LIBRARY
          PYTHON_VERSION_STRING
          Python3_EXECUTABLE
          Python3_INCLUDE_DIR
          Python3_LIBRARY
          Python_EXECUTABLE
          Python_INCLUDE_DIR
          Python_LIBRARY
          SKBUILD
      
      
      -- Build files have been written to: /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/_skbuild/linux-x86_64-3.10/cmake-build
      [1/64] Building CXX object lib/CMakeFiles/deepmd.dir/src/prod_env_mat_nvnmd.cc.o
      [2/64] Building CXX object op/CMakeFiles/op_abi.dir/map_nvnmd.cc.o
      FAILED: op/CMakeFiles/op_abi.dir/map_nvnmd.cc.o
      /bin/c++ -D_GLIBCXX_USE_CXX11_ABI=1 -Dop_abi_EXPORTS -I/home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include -I/nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/lib/include -fopenmp -O3 -DNDEBUG -fPIC -std=gnu++1y -MD -MT op/CMakeFiles/op_abi.dir/map_nvnmd.cc.o -MF op/CMakeFiles/op_abi.dir/map_nvnmd.cc.o.d -o op/CMakeFiles/op_abi.dir/map_nvnmd.cc.o -c /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/map_nvnmd.cc
      In file included from /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include/tensorflow/core/framework/op.h:23:0,
                       from /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/custom_op.h:6,
                       from /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/map_nvnmd.cc:26:
      /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include/tensorflow/core/framework/full_type.pb.h:10:40: fatal error: google/protobuf/port_def.inc: No such file or directory
       #include <google/protobuf/port_def.inc>
                                              ^
      compilation terminated.
      [3/64] Building CXX object op/CMakeFiles/op_abi.dir/descrpt.cc.o
      FAILED: op/CMakeFiles/op_abi.dir/descrpt.cc.o
      /bin/c++ -D_GLIBCXX_USE_CXX11_ABI=1 -Dop_abi_EXPORTS -I/home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include -I/nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/lib/include -fopenmp -O3 -DNDEBUG -fPIC -std=gnu++1y -MD -MT op/CMakeFiles/op_abi.dir/descrpt.cc.o -MF op/CMakeFiles/op_abi.dir/descrpt.cc.o.d -o op/CMakeFiles/op_abi.dir/descrpt.cc.o -c /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/descrpt.cc
      In file included from /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include/tensorflow/core/framework/op.h:23:0,
                       from /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/custom_op.h:6,
                       from /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/descrpt.cc:1:
      /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include/tensorflow/core/framework/full_type.pb.h:10:40: fatal error: google/protobuf/port_def.inc: No such file or directory
       #include <google/protobuf/port_def.inc>
                                              ^
      compilation terminated.
      [4/64] Building CXX object op/CMakeFiles/op_abi.dir/neighbor_stat.cc.o
      FAILED: op/CMakeFiles/op_abi.dir/neighbor_stat.cc.o
      /bin/c++ -D_GLIBCXX_USE_CXX11_ABI=1 -Dop_abi_EXPORTS -I/home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include -I/nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/lib/include -fopenmp -O3 -DNDEBUG -fPIC -std=gnu++1y -MD -MT op/CMakeFiles/op_abi.dir/neighbor_stat.cc.o -MF op/CMakeFiles/op_abi.dir/neighbor_stat.cc.o.d -o op/CMakeFiles/op_abi.dir/neighbor_stat.cc.o -c /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/neighbor_stat.cc
      In file included from /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include/tensorflow/core/framework/op.h:23:0,
                       from /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/custom_op.h:6,
                       from /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/neighbor_stat.cc:1:
      /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include/tensorflow/core/framework/full_type.pb.h:10:40: fatal error: google/protobuf/port_def.inc: No such file or directory
       #include <google/protobuf/port_def.inc>
                                              ^
      compilation terminated.
      [5/64] Building CXX object op/CMakeFiles/op_abi.dir/descrpt_se_a_ef_para.cc.o
      FAILED: op/CMakeFiles/op_abi.dir/descrpt_se_a_ef_para.cc.o
      /bin/c++ -D_GLIBCXX_USE_CXX11_ABI=1 -Dop_abi_EXPORTS -I/home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include -I/nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/lib/include -fopenmp -O3 -DNDEBUG -fPIC -std=gnu++1y -MD -MT op/CMakeFiles/op_abi.dir/descrpt_se_a_ef_para.cc.o -MF op/CMakeFiles/op_abi.dir/descrpt_se_a_ef_para.cc.o.d -o op/CMakeFiles/op_abi.dir/descrpt_se_a_ef_para.cc.o -c /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/descrpt_se_a_ef_para.cc
      In file included from /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include/tensorflow/core/framework/op.h:23:0,
                       from /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/custom_op.h:6,
                       from /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/descrpt_se_a_ef_para.cc:1:
      /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include/tensorflow/core/framework/full_type.pb.h:10:40: fatal error: google/protobuf/port_def.inc: No such file or directory
       #include <google/protobuf/port_def.inc>
                                              ^
      compilation terminated.
      [6/64] Building CXX object op/CMakeFiles/op_abi.dir/ewald_recp.cc.o
      FAILED: op/CMakeFiles/op_abi.dir/ewald_recp.cc.o
      /bin/c++ -D_GLIBCXX_USE_CXX11_ABI=1 -Dop_abi_EXPORTS -I/home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include -I/nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/lib/include -fopenmp -O3 -DNDEBUG -fPIC -std=gnu++1y -MD -MT op/CMakeFiles/op_abi.dir/ewald_recp.cc.o -MF op/CMakeFiles/op_abi.dir/ewald_recp.cc.o.d -o op/CMakeFiles/op_abi.dir/ewald_recp.cc.o -c /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/ewald_recp.cc
      In file included from /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include/tensorflow/core/framework/op.h:23:0,
                       from /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/custom_op.h:6,
                       from /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/ewald_recp.cc:1:
      /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include/tensorflow/core/framework/full_type.pb.h:10:40: fatal error: google/protobuf/port_def.inc: No such file or directory
       #include <google/protobuf/port_def.inc>
                                              ^
      compilation terminated.
      [7/64] Building CXX object op/CMakeFiles/op_abi.dir/prod_virial_multi_device.cc.o
      FAILED: op/CMakeFiles/op_abi.dir/prod_virial_multi_device.cc.o
      /bin/c++ -D_GLIBCXX_USE_CXX11_ABI=1 -Dop_abi_EXPORTS -I/home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include -I/nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/lib/include -fopenmp -O3 -DNDEBUG -fPIC -std=gnu++1y -MD -MT op/CMakeFiles/op_abi.dir/prod_virial_multi_device.cc.o -MF op/CMakeFiles/op_abi.dir/prod_virial_multi_device.cc.o.d -o op/CMakeFiles/op_abi.dir/prod_virial_multi_device.cc.o -c /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/prod_virial_multi_device.cc
      In file included from /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include/tensorflow/core/framework/op.h:23:0,
                       from /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/custom_op.h:6,
                       from /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/prod_virial_multi_device.cc:1:
      /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include/tensorflow/core/framework/full_type.pb.h:10:40: fatal error: google/protobuf/port_def.inc: No such file or directory
       #include <google/protobuf/port_def.inc>
                                              ^
      compilation terminated.
      [8/64] Building CXX object op/CMakeFiles/op_abi.dir/prod_virial.cc.o
      FAILED: op/CMakeFiles/op_abi.dir/prod_virial.cc.o
      /bin/c++ -D_GLIBCXX_USE_CXX11_ABI=1 -Dop_abi_EXPORTS -I/home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include -I/nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/lib/include -fopenmp -O3 -DNDEBUG -fPIC -std=gnu++1y -MD -MT op/CMakeFiles/op_abi.dir/prod_virial.cc.o -MF op/CMakeFiles/op_abi.dir/prod_virial.cc.o.d -o op/CMakeFiles/op_abi.dir/prod_virial.cc.o -c /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/prod_virial.cc
      In file included from /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include/tensorflow/core/framework/op.h:23:0,
                       from /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/custom_op.h:6,
                       from /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/prod_virial.cc:1:
      /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include/tensorflow/core/framework/full_type.pb.h:10:40: fatal error: google/protobuf/port_def.inc: No such file or directory
       #include <google/protobuf/port_def.inc>
                                              ^
      compilation terminated.
      [9/64] Building CXX object op/CMakeFiles/op_abi.dir/soft_min.cc.o
      FAILED: op/CMakeFiles/op_abi.dir/soft_min.cc.o
      /bin/c++ -D_GLIBCXX_USE_CXX11_ABI=1 -Dop_abi_EXPORTS -I/home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include -I/nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/lib/include -fopenmp -O3 -DNDEBUG -fPIC -std=gnu++1y -MD -MT op/CMakeFiles/op_abi.dir/soft_min.cc.o -MF op/CMakeFiles/op_abi.dir/soft_min.cc.o.d -o op/CMakeFiles/op_abi.dir/soft_min.cc.o -c /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/soft_min.cc
      In file included from /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include/tensorflow/core/framework/op.h:23:0,
                       from /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/custom_op.h:6,
                       from /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/soft_min.cc:1:
      /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include/tensorflow/core/framework/full_type.pb.h:10:40: fatal error: google/protobuf/port_def.inc: No such file or directory
       #include <google/protobuf/port_def.inc>
                                              ^
      compilation terminated.
      [10/64] Building CXX object op/CMakeFiles/op_abi.dir/soft_min_force.cc.o
      FAILED: op/CMakeFiles/op_abi.dir/soft_min_force.cc.o
      /bin/c++ -D_GLIBCXX_USE_CXX11_ABI=1 -Dop_abi_EXPORTS -I/home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include -I/nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/lib/include -fopenmp -O3 -DNDEBUG -fPIC -std=gnu++1y -MD -MT op/CMakeFiles/op_abi.dir/soft_min_force.cc.o -MF op/CMakeFiles/op_abi.dir/soft_min_force.cc.o.d -o op/CMakeFiles/op_abi.dir/soft_min_force.cc.o -c /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/soft_min_force.cc
      In file included from /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include/tensorflow/core/framework/op.h:23:0,
                       from /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/custom_op.h:6,
                       from /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/soft_min_force.cc:1:
      /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include/tensorflow/core/framework/full_type.pb.h:10:40: fatal error: google/protobuf/port_def.inc: No such file or directory
       #include <google/protobuf/port_def.inc>
                                              ^
      compilation terminated.
      [11/64] Building CXX object op/CMakeFiles/op_abi.dir/map_aparam.cc.o
      FAILED: op/CMakeFiles/op_abi.dir/map_aparam.cc.o
      /bin/c++ -D_GLIBCXX_USE_CXX11_ABI=1 -Dop_abi_EXPORTS -I/home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include -I/nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/lib/include -fopenmp -O3 -DNDEBUG -fPIC -std=gnu++1y -MD -MT op/CMakeFiles/op_abi.dir/map_aparam.cc.o -MF op/CMakeFiles/op_abi.dir/map_aparam.cc.o.d -o op/CMakeFiles/op_abi.dir/map_aparam.cc.o -c /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/map_aparam.cc
      In file included from /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include/tensorflow/core/framework/op.h:23:0,
                       from /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/custom_op.h:6,
                       from /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/map_aparam.cc:1:
      /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include/tensorflow/core/framework/full_type.pb.h:10:40: fatal error: google/protobuf/port_def.inc: No such file or directory
       #include <google/protobuf/port_def.inc>
                                              ^
      compilation terminated.
      [12/64] Building CXX object op/CMakeFiles/op_abi.dir/prod_force.cc.o
      FAILED: op/CMakeFiles/op_abi.dir/prod_force.cc.o
      /bin/c++ -D_GLIBCXX_USE_CXX11_ABI=1 -Dop_abi_EXPORTS -I/home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include -I/nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/lib/include -fopenmp -O3 -DNDEBUG -fPIC -std=gnu++1y -MD -MT op/CMakeFiles/op_abi.dir/prod_force.cc.o -MF op/CMakeFiles/op_abi.dir/prod_force.cc.o.d -o op/CMakeFiles/op_abi.dir/prod_force.cc.o -c /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/prod_force.cc
      In file included from /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include/tensorflow/core/framework/op.h:23:0,
                       from /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/custom_op.h:6,
                       from /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/prod_force.cc:1:
      /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include/tensorflow/core/framework/full_type.pb.h:10:40: fatal error: google/protobuf/port_def.inc: No such file or directory
       #include <google/protobuf/port_def.inc>
                                              ^
      compilation terminated.
      [13/64] Building CXX object op/CMakeFiles/op_abi.dir/quantize_nvnmd.cc.o
      FAILED: op/CMakeFiles/op_abi.dir/quantize_nvnmd.cc.o
      /bin/c++ -D_GLIBCXX_USE_CXX11_ABI=1 -Dop_abi_EXPORTS -I/home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include -I/nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/lib/include -fopenmp -O3 -DNDEBUG -fPIC -std=gnu++1y -MD -MT op/CMakeFiles/op_abi.dir/quantize_nvnmd.cc.o -MF op/CMakeFiles/op_abi.dir/quantize_nvnmd.cc.o.d -o op/CMakeFiles/op_abi.dir/quantize_nvnmd.cc.o -c /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/quantize_nvnmd.cc
      In file included from /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include/tensorflow/core/framework/op.h:23:0,
                       from /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/custom_op.h:6,
                       from /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/quantize_nvnmd.cc:26:
      /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include/tensorflow/core/framework/full_type.pb.h:10:40: fatal error: google/protobuf/port_def.inc: No such file or directory
       #include <google/protobuf/port_def.inc>
                                              ^
      compilation terminated.
      [14/64] Building CXX object op/CMakeFiles/op_abi.dir/pair_tab.cc.o
      FAILED: op/CMakeFiles/op_abi.dir/pair_tab.cc.o
      /bin/c++ -D_GLIBCXX_USE_CXX11_ABI=1 -Dop_abi_EXPORTS -I/home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include -I/nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/lib/include -fopenmp -O3 -DNDEBUG -fPIC -std=gnu++1y -MD -MT op/CMakeFiles/op_abi.dir/pair_tab.cc.o -MF op/CMakeFiles/op_abi.dir/pair_tab.cc.o.d -o op/CMakeFiles/op_abi.dir/pair_tab.cc.o -c /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/pair_tab.cc
      In file included from /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include/tensorflow/core/framework/op.h:23:0,
                       from /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/custom_op.h:6,
                       from /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/pair_tab.cc:1:
      /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include/tensorflow/core/framework/full_type.pb.h:10:40: fatal error: google/protobuf/port_def.inc: No such file or directory
       #include <google/protobuf/port_def.inc>
                                              ^
      compilation terminated.
      [15/64] Building CXX object op/CMakeFiles/op_abi.dir/prod_force_multi_device.cc.o
      FAILED: op/CMakeFiles/op_abi.dir/prod_force_multi_device.cc.o
      /bin/c++ -D_GLIBCXX_USE_CXX11_ABI=1 -Dop_abi_EXPORTS -I/home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include -I/nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/lib/include -fopenmp -O3 -DNDEBUG -fPIC -std=gnu++1y -MD -MT op/CMakeFiles/op_abi.dir/prod_force_multi_device.cc.o -MF op/CMakeFiles/op_abi.dir/prod_force_multi_device.cc.o.d -o op/CMakeFiles/op_abi.dir/prod_force_multi_device.cc.o -c /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/prod_force_multi_device.cc
      In file included from /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include/tensorflow/core/framework/op.h:23:0,
                       from /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/custom_op.h:6,
                       from /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/prod_force_multi_device.cc:1:
      /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include/tensorflow/core/framework/full_type.pb.h:10:40: fatal error: google/protobuf/port_def.inc: No such file or directory
       #include <google/protobuf/port_def.inc>
                                              ^
      compilation terminated.
      [16/64] Building CXX object op/CMakeFiles/op_abi.dir/matmul_nvnmd.cc.o
      FAILED: op/CMakeFiles/op_abi.dir/matmul_nvnmd.cc.o
      /bin/c++ -D_GLIBCXX_USE_CXX11_ABI=1 -Dop_abi_EXPORTS -I/home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include -I/nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/lib/include -fopenmp -O3 -DNDEBUG -fPIC -std=gnu++1y -MD -MT op/CMakeFiles/op_abi.dir/matmul_nvnmd.cc.o -MF op/CMakeFiles/op_abi.dir/matmul_nvnmd.cc.o.d -o op/CMakeFiles/op_abi.dir/matmul_nvnmd.cc.o -c /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/matmul_nvnmd.cc
      In file included from /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include/tensorflow/core/framework/op.h:23:0,
                       from /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/custom_op.h:6,
                       from /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/matmul_nvnmd.cc:25:
      /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include/tensorflow/core/framework/full_type.pb.h:10:40: fatal error: google/protobuf/port_def.inc: No such file or directory
       #include <google/protobuf/port_def.inc>
                                              ^
      compilation terminated.
      [17/64] Building CXX object op/CMakeFiles/op_abi.dir/soft_min_virial.cc.o
      FAILED: op/CMakeFiles/op_abi.dir/soft_min_virial.cc.o
      /bin/c++ -D_GLIBCXX_USE_CXX11_ABI=1 -Dop_abi_EXPORTS -I/home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include -I/nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/lib/include -fopenmp -O3 -DNDEBUG -fPIC -std=gnu++1y -MD -MT op/CMakeFiles/op_abi.dir/soft_min_virial.cc.o -MF op/CMakeFiles/op_abi.dir/soft_min_virial.cc.o.d -o op/CMakeFiles/op_abi.dir/soft_min_virial.cc.o -c /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/soft_min_virial.cc
      In file included from /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include/tensorflow/core/framework/op.h:23:0,
                       from /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/custom_op.h:6,
                       from /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/soft_min_virial.cc:1:
      /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include/tensorflow/core/framework/full_type.pb.h:10:40: fatal error: google/protobuf/port_def.inc: No such file or directory
       #include <google/protobuf/port_def.inc>
                                              ^
      compilation terminated.
      [18/64] Building CXX object op/CMakeFiles/op_abi.dir/descrpt_se_a_ef_vert.cc.o
      FAILED: op/CMakeFiles/op_abi.dir/descrpt_se_a_ef_vert.cc.o
      /bin/c++ -D_GLIBCXX_USE_CXX11_ABI=1 -Dop_abi_EXPORTS -I/home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include -I/nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/lib/include -fopenmp -O3 -DNDEBUG -fPIC -std=gnu++1y -MD -MT op/CMakeFiles/op_abi.dir/descrpt_se_a_ef_vert.cc.o -MF op/CMakeFiles/op_abi.dir/descrpt_se_a_ef_vert.cc.o.d -o op/CMakeFiles/op_abi.dir/descrpt_se_a_ef_vert.cc.o -c /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/descrpt_se_a_ef_vert.cc
      In file included from /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include/tensorflow/core/framework/op.h:23:0,
                       from /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/custom_op.h:6,
                       from /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/descrpt_se_a_ef_vert.cc:1:
      /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include/tensorflow/core/framework/full_type.pb.h:10:40: fatal error: google/protobuf/port_def.inc: No such file or directory
       #include <google/protobuf/port_def.inc>
                                              ^
      compilation terminated.
      [19/64] Building CXX object op/CMakeFiles/op_abi.dir/prod_env_mat_multi_device_nvnmd.cc.o
      FAILED: op/CMakeFiles/op_abi.dir/prod_env_mat_multi_device_nvnmd.cc.o
      /bin/c++ -D_GLIBCXX_USE_CXX11_ABI=1 -Dop_abi_EXPORTS -I/home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include -I/nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/lib/include -fopenmp -O3 -DNDEBUG -fPIC -std=gnu++1y -MD -MT op/CMakeFiles/op_abi.dir/prod_env_mat_multi_device_nvnmd.cc.o -MF op/CMakeFiles/op_abi.dir/prod_env_mat_multi_device_nvnmd.cc.o.d -o op/CMakeFiles/op_abi.dir/prod_env_mat_multi_device_nvnmd.cc.o -c /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/prod_env_mat_multi_device_nvnmd.cc
      In file included from /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include/tensorflow/core/framework/op.h:23:0,
                       from /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/custom_op.h:6,
                       from /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/prod_env_mat_multi_device_nvnmd.cc:19:
      /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include/tensorflow/core/framework/full_type.pb.h:10:40: fatal error: google/protobuf/port_def.inc: No such file or directory
       #include <google/protobuf/port_def.inc>
                                              ^
      compilation terminated.
      [20/64] Building CXX object op/CMakeFiles/op_abi.dir/prod_env_mat_multi_device.cc.o
      FAILED: op/CMakeFiles/op_abi.dir/prod_env_mat_multi_device.cc.o
      /bin/c++ -D_GLIBCXX_USE_CXX11_ABI=1 -Dop_abi_EXPORTS -I/home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include -I/nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/lib/include -fopenmp -O3 -DNDEBUG -fPIC -std=gnu++1y -MD -MT op/CMakeFiles/op_abi.dir/prod_env_mat_multi_device.cc.o -MF op/CMakeFiles/op_abi.dir/prod_env_mat_multi_device.cc.o.d -o op/CMakeFiles/op_abi.dir/prod_env_mat_multi_device.cc.o -c /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/prod_env_mat_multi_device.cc
      In file included from /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include/tensorflow/core/framework/op.h:23:0,
                       from /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/custom_op.h:6,
                       from /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/prod_env_mat_multi_device.cc:1:
      /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include/tensorflow/core/framework/full_type.pb.h:10:40: fatal error: google/protobuf/port_def.inc: No such file or directory
       #include <google/protobuf/port_def.inc>
                                              ^
      compilation terminated.
      [21/64] Building CXX object op/CMakeFiles/op_abi.dir/descrpt_se_a_ef.cc.o
      FAILED: op/CMakeFiles/op_abi.dir/descrpt_se_a_ef.cc.o
      /bin/c++ -D_GLIBCXX_USE_CXX11_ABI=1 -Dop_abi_EXPORTS -I/home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include -I/nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/lib/include -fopenmp -O3 -DNDEBUG -fPIC -std=gnu++1y -MD -MT op/CMakeFiles/op_abi.dir/descrpt_se_a_ef.cc.o -MF op/CMakeFiles/op_abi.dir/descrpt_se_a_ef.cc.o.d -o op/CMakeFiles/op_abi.dir/descrpt_se_a_ef.cc.o -c /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/descrpt_se_a_ef.cc
      In file included from /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include/tensorflow/core/framework/op.h:23:0,
                       from /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/custom_op.h:6,
                       from /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/descrpt_se_a_ef.cc:2:
      /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include/tensorflow/core/framework/full_type.pb.h:10:40: fatal error: google/protobuf/port_def.inc: No such file or directory
       #include <google/protobuf/port_def.inc>
                                              ^
      compilation terminated.
      [22/64] Building CXX object op/CMakeFiles/op_abi.dir/gelu_multi_device.cc.o
      FAILED: op/CMakeFiles/op_abi.dir/gelu_multi_device.cc.o
      /bin/c++ -D_GLIBCXX_USE_CXX11_ABI=1 -Dop_abi_EXPORTS -I/home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include -I/nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/lib/include -fopenmp -O3 -DNDEBUG -fPIC -std=gnu++1y -MD -MT op/CMakeFiles/op_abi.dir/gelu_multi_device.cc.o -MF op/CMakeFiles/op_abi.dir/gelu_multi_device.cc.o.d -o op/CMakeFiles/op_abi.dir/gelu_multi_device.cc.o -c /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/gelu_multi_device.cc
      In file included from /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include/tensorflow/core/framework/op.h:23:0,
                       from /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/custom_op.h:6,
                       from /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/gelu_multi_device.cc:1:
      /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include/tensorflow/core/framework/full_type.pb.h:10:40: fatal error: google/protobuf/port_def.inc: No such file or directory
       #include <google/protobuf/port_def.inc>
                                              ^
      compilation terminated.
      [23/64] Building CXX object op/CMakeFiles/op_abi.dir/custom_op.cc.o
      FAILED: op/CMakeFiles/op_abi.dir/custom_op.cc.o
      /bin/c++ -D_GLIBCXX_USE_CXX11_ABI=1 -Dop_abi_EXPORTS -I/home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include -I/nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/lib/include -fopenmp -O3 -DNDEBUG -fPIC -std=gnu++1y -MD -MT op/CMakeFiles/op_abi.dir/custom_op.cc.o -MF op/CMakeFiles/op_abi.dir/custom_op.cc.o.d -o op/CMakeFiles/op_abi.dir/custom_op.cc.o -c /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/custom_op.cc
      In file included from /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include/tensorflow/core/framework/op.h:23:0,
                       from /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/custom_op.h:6,
                       from /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source/op/custom_op.cc:1:
      /home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow/include/tensorflow/core/framework/full_type.pb.h:10:40: fatal error: google/protobuf/port_def.inc: No such file or directory
       #include <google/protobuf/port_def.inc>
                                              ^
      compilation terminated.
      ninja: build stopped: subcommand failed.
        File "/tmp/pip-build-env-5fs70bk8/overlay/lib/python3.10/site-packages/skbuild/setuptools_wrap.py", line 645, in setup
          cmkr.make(make_args, install_target=cmake_install_target, env=env)
        File "/tmp/pip-build-env-5fs70bk8/overlay/lib/python3.10/site-packages/skbuild/cmaker.py", line 680, in make
          self.make_impl(clargs=clargs, config=config, source_dir=source_dir, install_target=install_target, env=env)
        File "/tmp/pip-build-env-5fs70bk8/overlay/lib/python3.10/site-packages/skbuild/cmaker.py", line 704, in make_impl
          raise SKBuildError(
      
      
      --------------------------------------------------------------------------------
      -- Trying "Ninja" generator
      --------------------------------
      ---------------------------
      ----------------------
      -----------------
      ------------
      -------
      --
      --
      -------
      ------------
      -----------------
      ----------------------
      ---------------------------
      --------------------------------
      -- Trying "Ninja" generator - success
      --------------------------------------------------------------------------------
      
      Configuring Project
        Working directory:
          /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/_skbuild/linux-x86_64-3.10/cmake-build
        Command:
          cmake /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/source -G Ninja -DCMAKE_INSTALL_PREFIX:PATH=/nobackup1c/users/zhenzey/DeepMD/deepmd-kit/_skbuild/linux-x86_64-3.10/cmake-install -DPYTHON_VERSION_STRING:STRING=3.10.4 -DSKBUILD:INTERNAL=TRUE -DCMAKE_MODULE_PATH:PATH=/tmp/pip-build-env-5fs70bk8/overlay/lib/python3.10/site-packages/skbuild/resources/cmake -DPython3_EXECUTABLE:FILEPATH=/home/zhenzey/.conda/envs/deepmd/bin/python -DPython3_INCLUDE_DIR:PATH=/home/zhenzey/.conda/envs/deepmd/include/python3.10 -DPython3_LIBRARY:PATH=/home/zhenzey/.conda/envs/deepmd/lib/libpython3.10.so -DPython_EXECUTABLE:FILEPATH=/home/zhenzey/.conda/envs/deepmd/bin/python -DPython_INCLUDE_DIR:PATH=/home/zhenzey/.conda/envs/deepmd/include/python3.10 -DPython_LIBRARY:PATH=/home/zhenzey/.conda/envs/deepmd/lib/libpython3.10.so -DPYTHON_EXECUTABLE:FILEPATH=/home/zhenzey/.conda/envs/deepmd/bin/python -DPYTHON_INCLUDE_DIR:PATH=/home/zhenzey/.conda/envs/deepmd/include/python3.10 -DPYTHON_LIBRARY:PATH=/home/zhenzey/.conda/envs/deepmd/lib/libpython3.10.so -DCMAKE_MAKE_PROGRAM:FILEPATH=/tmp/pip-build-env-5fs70bk8/overlay/lib/python3.10/site-packages/ninja/data/bin/ninja -DTENSORFLOW_ROOT:STRING=/home/zhenzey/.conda/envs/deepmd/lib/python3.10/site-packages/tensorflow -DBUILD_PY_IF:BOOL=TRUE -DBUILD_CPP_IF:BOOL=FALSE -DCMAKE_BUILD_TYPE:STRING=Release
      
      Traceback (most recent call last):
      
      An error occurred while building with CMake.
        Command:
          cmake --build . --target install --config Release --
        Install target:
          install
        Source directory:
          /nobackup1c/users/zhenzey/DeepMD/deepmd-kit
        Working directory:
          /nobackup1c/users/zhenzey/DeepMD/deepmd-kit/_skbuild/linux-x86_64-3.10/cmake-build
      Please check the install target is valid and see CMake's output for more information.
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for deepmd-kit
Failed to build deepmd-kit
ERROR: Could not build wheels for deepmd-kit, which is required to install pyproject.toml-based projects

--- a/source/cmake/Findtensorflow.cmake
+++ b/source/cmake/Findtensorflow.cmake
@@ -47,11 +47,17 @@ endif(DEFINED TENSORFLOW_ROOT)
 
 # define the search path
 list(APPEND TensorFlow_search_PATHS ${TENSORFLOW_ROOT})
-list(APPEND TensorFlow_search_PATHS "${TENSORFLOW_ROOT}/../tensorflow_core")
+if(BUILD_CPP_IF)
 list(APPEND TensorFlow_search_PATHS ${TENSORFLOW_ROOT_NO64})
-list(APPEND TensorFlow_search_PATHS "${TENSORFLOW_ROOT_NO64}/../tensorflow_core")
 list(APPEND TensorFlow_search_PATHS "/usr/")
 list(APPEND TensorFlow_search_PATHS "/usr/local/")
+endif()
+if(BUILD_PY_IF)
+	# here TENSORFLOW_ROOT is path to site-packages/tensorflow
+	# for conda libraries, append extra paths
+	list(APPEND TensorFlow_search_PATHS "${TENSORFLOW_ROOT}/../tensorflow_core")
+	list(APPEND TensorFlow_search_PATHS "${TENSORFLOW_ROOT}/../../../..")
+endif()
 
 # includes
 find_path(TensorFlow_INCLUDE_DIRS
@@ -65,7 +71,6 @@ find_path(TensorFlow_INCLUDE_DIRS
   PATH_SUFFIXES "/include"
   NO_DEFAULT_PATH
   )
-if (BUILD_CPP_IF)
 find_path(TensorFlow_INCLUDE_DIRS_GOOGLE
   NAMES 
   google/protobuf/type.pb.h
@@ -74,11 +79,10 @@ find_path(TensorFlow_INCLUDE_DIRS_GOOGLE
   NO_DEFAULT_PATH
   )
 list(APPEND TensorFlow_INCLUDE_DIRS ${TensorFlow_INCLUDE_DIRS_GOOGLE})
-endif ()
   
 if (NOT TensorFlow_INCLUDE_DIRS AND tensorflow_FIND_REQUIRED)
   message(FATAL_ERROR 
-    "Not found 'tensorflow/core/public/session.h' directory in path '${TensorFlow_search_PATHS}' "
+    "Not found 'include/tensorflow/core/public/session.h' directory or other header files in path '${TensorFlow_search_PATHS}' "
     "You can manually set the tensorflow install path by -DTENSORFLOW_ROOT ")
 endif ()
 

